setup.py

```diff
@@ -16,7 +16,6 @@
 
 import setuptools
 
-
-setuptools.setup(
-    setup_requires=['pbr>=2.0.0'],
-    pbr=True)
+setuptools.setup(setup_requires=['pbr>=2.0.0'],
+                 pbr=True,
+                 data_files=['ChangeLog'])
```

Comment:
@theatul This is to add ChangeLog in .whl file under data dir. This _**seems**_ optional to me.